app/src/main/java/de/tum/in/tumcampusapp/component/tumui/feedback/FeedbackPresenter.kt

```diff
@@ -127,8 +125,11 @@ class FeedbackPresenter @Inject constructor(
         val authority = "de.tum.in.tumcampusapp.fileprovider"
         val photoURI = FileProvider.getUriForFile(context, authority, photoFile)
         takePictureIntent.putExtra(MediaStore.EXTRA_OUTPUT, photoURI)
-
-        view?.openCamera(takePictureIntent)
+        try {
+            view?.openCamera(takePictureIntent)
+        } catch (e: ActivityNotFoundException) {
+            photoFile.delete()
```

Comment:
why is this required? is the file already created when getting the Uri?

Reply:
The file is created before trying to open the new activity (line 115). If the activity cannot be opened, the file won't be used. Therefore, it should be deleted again. Previously, the file would not even be created if the check which I have removed failed.

Reply:
Ah sorry, did not open the collapsed section :D